src/core_modules/capture-core/components/Pages/NewRelationship/RegisterTei/GeneralOutput/WarningsSection/SearchGroupDuplicate/ReviewDialogContentsPager.component.js

```diff
@@ -15,24 +16,17 @@ const getStyles = (theme: Theme) => ({
     },
 });
 
-type Props = {
-    paginationData: Object,
-    onChangePage: Function,
-    classes: Object,
-};
+const ReviewDialogContentsPagerPlain = ({ onChangePage, currentPage, nextPageButtonDisabled, classes }: Props) => (
+    <div
+        className={classes.container}
+    >
+        <Pager
+            currentPage={currentPage}
+            onChangePage={page => onChangePage(page)}
+            nextPageButtonDisabled={nextPageButtonDisabled}
+        />
+    </div>
+);
 
-const ReviewDialogContentsPager = (props: Props) => {
-    const { onChangePage, paginationData, classes } = props;
-    return (
-        <div
-            className={classes.container}
-        >
-            <Pager
-                onChangePage={onChangePage}
-                {...paginationData}
-            />
-        </div>
-    );
-};
-
-export default withStyles(getStyles)(ReviewDialogContentsPager);
+export const ReviewDialogContentsPagerComponent: ComponentType<$Diff<Props, CssClasses>> =
```

Comment:
👍 Great, I like this approach better.

Reply:
Yeah me too

Reply:
No basically when we use the component we can see which props it takes and all of those stuff